### ISSUE TITLE
Suggestion: Powershell Execution Policy Bypass added to 051-Data-Marc_ModelDocumentor.pbitool.json

### ISSUE THREAD

**Issue by paul-rami** [marclelijveld/External-Tools-Model-Documentation#19]
Hi Marc,

First of all thanks for this amazing tool! 

Just ran into the 3rd issue described on the FAQ, where the PowerShell window pops up but no action happens afterward.

I am not a developer so I don't feel comfortable playing with the PowerSheel configuration, and wondered if adding the following line (see image below) would work in bypassing the execution policies? As you see just picked the idea from another external tool that works with my current setup (Hot Swap Connections).

Thanks again for your time!

![image](https://user-images.githubusercontent.com/68693228/121874663-dbbd1800-cd07-11eb-826f-491c9d8abf2c.png)

**Comment by paul-rami** [comment]
Hi Marc,  I tried adding that line to "051-Data-Marc_ModelDocumentor.pbitool.json" with no luck. Do you have any recommendation that does not involve digging too deep in PowerShell policies?

**Comment by marclelijveld** [comment]
Hi @paul-rami , 

Thanks for bringing this to my attention and my apologies for this horrible late reply! 
I see you explored other options to bypass the execution policies. Did you get this to work? 
On my end, I will reach out to the developer of hot swap connections and ask how exactly this works, so I can potentially include this in the future version of the tool. 

Thanks for now! 
--Marc

**Comment by marclelijveld** [comment]
Hi @paul-rami, 

I have been in contact with Steve Campbell. He is the developer of the Hot Swap Connections external tool. He told me that it is due to a change they made for bus ops tool - they move the location of the powershell scripts. As a result, the filename broke as it had spaces. This was fixed on the new version of bus ops and I made sure your tool was too.

**How to fix:**
- download latest version of business ops
- Click "Edit External tools" and remove any powershell tools:
    - Document Model
    - Hot Swap
    - Clear Cache
- Add them back using business ops

Kudos for this solution all to @PowerBISteve  

In the mean time I'll work on adding this to the tool natively. Even if it is installed from outside BusinessOps. 

Cheers, 
Marc
